frontend/packages/dev-console/src/components/dropdown/ResourceDropdown.tsx

```diff
@@ -159,8 +159,12 @@ class ResourceDropdown extends React.Component<ResourceDropdownProps, State> {
 
   private handleChange = (key, items) => {
     const name = items[key];
-    const { actionItem, onChange, selectedKey } = this.props;
-    const title = actionItem && key === actionItem.actionKey ? actionItem.actionTitle : name;
+    const { actionItems, onChange, selectedKey } = this.props;
+    const selectedActionItem = actionItems && actionItems.filter((ai) => key === ai.actionKey);
```

Comment:
You could use `#find` to locate the selected item in the `actionItems` array.

Reply:
@christianvogt fixed in 20bf54c